src/styles/util.scss

```diff
@@ -2,6 +2,8 @@
 @use './mixins/utils';
 @use './variables' as var;
 
+@tailwind utilities;
+
```

Comment:
This likely wouldn't go here, exactly, but it's what generates the utility classes.